src/Contacts/include/BipedalLocomotion/Contacts/ContactListParser.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file ContactListParser.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_CONTACTS_CONTACTLIST_PARSER_H
+#define BIPEDAL_LOCOMOTION_CONTACTS_CONTACTLIST_PARSER_H
```

Comment:
I don't know if it is worth adding "File" or "Json" also in the name of this file.

Reply:
File renamed in `ContactListJsonParser.h`